connector/wrappers/i2b2/query_test.go

```diff
@@ -37,19 +52,33 @@ func TestGetOntologyConceptChildren(t *testing.T) {
 	if err != nil {
 		t.Fail()
 	}
+
+	t.Log("Length of found results array ", len(results))
+	fmt.Println("Found length of results ", len(results))
+	for _, r := range results {
+		t.Log("Found result ", r.DisplayName)
+		fmt.Println("Found result: ", r.DisplayName)
+	}
+	//TODO add a test to verify that there are 3 children concepts to e2etest
 	t.Log(*results[0].MedcoEncryption)
 }
 
 func TestGetOntologyModifiers(t *testing.T) {
+	//In order to prepare the queries to the sql database done inside GetOntologyModifiers we need to call the "test database" functions
+	utilserver.TestDBConnection(t)
+	utilserver.TestI2B2DBConnection(t)
```

Comment:
```
	utilserver.TestDBConnection(t)
	utilserver.TestI2B2DBConnection(t)
```
why that? (also below)

Reply:
This is due to the fact that fetching ontology modifiers using `GetOntologyModifiers` now calls function (added by this PR) that perform some SQL queries to fetch modifiers subject counts (which is not done by default by I2B2). If I recall correctly in golang unit tests where SQL queries are performed (even it is under the hood), you need to call
```golang
	utilserver.TestDBConnection(t)
	utilserver.TestI2B2DBConnection(t)
```
as a prerequisite so that the code called by the test is able to perform SQL queries.